helper_libs/og_themes.py

```
@@ -0,0 +1,615 @@
+                subtitle_y += int(theme.subtitle_font_size * 1.4)
+        
+        return img
+    
+    def _wrap_text(self, text: str, font: ImageFont.FreeTypeFont, max_width: int) -> List[str]:
+        """Wrap text to fit within max_width."""
+        words = text.split()
+        lines = []
+        current_line = []
+        
+        for word in words:
+            test_line = ' '.join(current_line + [word])
```

Ruff (0.14.10)

Consider `[*current_line, word]` instead of concatenation

Replace with `[*current_line, word]`

(RUF005)